tests/backend/user/test_apiv1_user_invite.py

```diff
@@ -67,7 +68,9 @@ async def test_user_invite_timeout(mock_clock, backend, apiv1_alice_backend_sock
     with backend.event_bus.listen() as spy:
         async with user_invite(apiv1_alice_backend_sock, user_id=mallory.user_id) as prep:
 
-            await spy.wait_with_timeout("event.connected", {"event_name": "user.claimed"})
+            await spy.wait_with_timeout(
+                "event.connected", {"event_name": BackendEvent.user_claimed}
```

Comment:
We should also have a `EventBusMetaEvent` Enum (or a better/shorter name 😄 ) with EVENT_CONNECTED/EVENT_DISCONNECTED.
This would allow to add proper typing to the `EventBus` class and remove this dirty check:
https://github.com/Scille/parsec-cloud/blob/335bdb699684f593454320ba53d63f7501ea95c0/parsec/event_bus.py#L47-L49

Reply:
also we could rename `event_name` param into `event_type` (calling it `name` make is feel like it's a string...)

Reply:
I added a new Enum in `parsec.event_bus`:
```python
class MetaEvent(Enum):
    EVENT_CONNECTED = "event.connected"
    EVENT_DISCONNECTED = "event.disconnected"
```

`MetaEvent` is a name like `CoreEvent` 


I also renamed `event_name` param into `event_type`